package.json

```diff
@@ -27,6 +30,8 @@
     "eslint": "^6.6.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
-    "jest": "^24.9.0"
+    "jest": "^24.9.0",
+    "stylelint": "^11.1.1",
+    "stylelint-webpack-plugin": "^1.0.3"
```

Comment:
Why this dep?

Reply:
Sorry, this was a mistake. I thought we need both stylelint and stylelint-webpack-plugin for stylelint to work. I'll remove it